src/resources/animation.js

```diff
@@ -6,9 +6,16 @@ Object.assign(pc, function () {
 
     Object.assign(AnimationHandler.prototype, {
         load: function (url, callback) {
-            pc.http.get(url, function (err, response) {
+            if (typeof url === 'string') {

```

Comment:
Do you need to allocate an object here? Perhaps instead of creating a new object you can check if url is already an object and create two variables from it like urlLoad and urlOriginal so you don't have to allocate it. Like 
```
var urlLoad = url;
var urlOriginal = url;
if (typeof url === 'object') {
    urlLoad = url.load;
    urlOriginal = url.original;
}
```